TAP version 13
# Subtest: end end
    1..0
ok 1 - end end

not ok 2 - Error: test end() method called more than once
  ---
  {"message":"Error: test end() method called more than once","test":"end end"}
  ...

# Subtest: end then async end
    1..0
ok 3 - end then async end

# Subtest: double async end
    1..0
ok 4 - double async end

# Subtest: plan end
    1..1
    ok 1 - this is fine
ok 5 - plan end

# Subtest: plan then async end
    1..1
    ok 1 - this is fine
ok 6 - plan then async end

# Subtest: plan end end
    1..1
    ok 1 - this is fine
ok 7 - plan end end

not ok 8 - Error: test end() method called more than once
  ---
  {"message":"Error: test end() method called more than once","test":"end then async end"}
  ...

not ok 9 - Error: test end() method called more than once
  ---
  {"message":"Error: test end() method called more than once","test":"plan end end"}
  ...

not ok 10 - Error: test end() method called more than once
  ---
  {"message":"Error: test end() method called more than once","test":"double async end"}
  ...

1..10

